app/models/label/LabelTable.scala

```diff
@@ -375,10 +376,34 @@ object LabelTable {
       "label_type_value" -> labelMetadata.labelTypeValue,
       "severity" -> labelMetadata.severity,
       "temporary" -> labelMetadata.temporary,
+      //this one is different because we're pulling from the tags and label_tags databases
+      "tags" -> getTagsFromLabelId(labelMetadata.labelId),
       "description" -> labelMetadata.description
     )
   }
 
+  //seperate sql request for getting tags, gets all the tags that correspond to the label Id.
```

Comment:
For the comments above function definitions, could you change it to be in the format used above the function `countLabelsByUserId`? That is the format we are using throughout the back-end code. Although I realize that there is a bunch of code in `LabelTable.scala` that is old and does not have that format, but we are keeping all new code in the better format :+1: